test/pos.test.ts

```diff
@@ -285,6 +303,34 @@ describe('matched-positions-pug', async () => {
     `)
   })
 
+  test('attributify `><`', async () => {
+    const uno = createGenerator({
+      presets: [
+        presetAttributify(),
+        presetUno(),
+      ],
+      shortcuts: {
+        '<custom-shortcut': 'text-teal',
+      },
+    })
+
+    expect(await match(uno, '<div border></div><div <custom-shortcut></div>'))
```

Comment:
I don't the specification part. The only thing I can say is Nu Html Checker didn't like it https://validator.w3.org/nu/#cl1c5 But on the browser's side, it's just working. So, if the browser parser is fault-tolerant to it, so should we. You can think of it as a stability test. It's there not because we endorse it, it's there because we're capable of it.